lia-core/src/main/java/com/lithium/community/android/manager/LiSDKManager.java

```diff
@@ -342,5 +345,63 @@ public void onTokenRequestCompleted(@Nullable LiTokenResponse response, @Nullabl
             }
         });
     }
+
+    /**
+     * Logs out the user from the community for this device, clears all references of the current user for this device, at both backend and local storage
+     * @param context - instance of {@link Context} to access local cache and storage
+     * @param callback - instance of {@link com.lithium.community.android.callback.Callback to inform about the state of the logout operation,
+     * success() will be called if everything goes well, a necessary exception will be returned in failure(..) if something isn't done.
+     */
+    public void logout(@NonNull  Context context, @NonNull Callback<Void, Throwable> callback) {
+        LiCoreSDKUtils.checkNotNull(context, MessageConstants.wasNull("context"));
+        LiCoreSDKUtils.checkNotNull(callback, MessageConstants.wasNull("callback"));
+        if (!isUserLoggedIn()) {
+            callback.abort(new IllegalAccessException(context.getString(R.string.li_error_logout_user_not_looged_in)));
+            return;
+        }
+        LiDeviceTokenProvider provider = getLiDeviceTokenProvider();
+        String deviceId = null;
+        if (provider != null) {
+            deviceId = provider.getDeviceId();
+        }
+        LiClientRequestParams.LiLogoutRequestParams params = new LiClientRequestParams.LiLogoutRequestParams(context, deviceId);
+        try {
+            LiClient client = LiClientManager.getLogoutClient(params);
+            client.processAsync(new LogoutRequestCallback(context, callback));
+        } catch (LiRestResponseException lrre) {
+            lrre.printStackTrace();
+            callback.failure(lrre);
```

Comment:
Should it be `abort` here? The call was never made, it aborted because of an exception.